python_examples/luxembourg_example/luxembourg_inputs/luxembourg_initial_infect.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
```

Comment:
This file is missing a header to describe what it does - see older files for examples

Reply:
Changed